### PR TITLE
Add checks to PR template

### PR DESCRIPTION
**Pull request by kanyesthaker**
### Description of Changes
Add a basic manual testing flow to the PR template

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
 A quick sentence or bullet point for each point is sufficient -- this is to make it easy to understand changes to the codebase that your reviewer may not be familiar with.
 
-### Link to Issue
-Link the appropriate GitHub Issue for this PR. You can link an issue by adding `closes #issue-number` to the PR message body.
+## Testing Checklists
+If your change adds critical functionality that should be tested on every PR, edit ``pull_requeste_template.md`` and add it to the checklist.
+### Basic Author Checklist 
+The PR author should verify that the following are fully functional.
+- [ ] With 1 user:
+  - [ ] The world loads, ther user can move/look around, and physics behave properly
```
Comment (kanyesthaker) [inline .github/pull_request_template.md:10]:
```suggestion
  - [ ] The world loads, the user can move/look around, and physics behave properly
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
-### Link to Issue
-Link the appropriate GitHub Issue for this PR. You can link an issue by adding `closes #issue-number` to the PR message body.
+## Testing Checklists
+If your change adds critical functionality that should be tested on every PR, edit ``pull_requeste_template.md`` and add it to the checklist.
+### Basic Author Checklist 
+The PR author should verify that the following are fully functional.
+- [ ] With 1 user:
+  - [ ] The world loads, the user can move/look around, and physics behave properly
+  - [ ] User can pause their client, mute/unmute themselves, and change their name
+- [ ] With 2 users:
```
Comment (samjgorman) [inline .github/pull_request_template.md:12]: Should we explicitly test 3 users? Or too much friction to start? Have often seen bugs pop up at 3+ that weren't present at 2

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,9 +1,29 @@
...
+  - [ ] A second user can join the same room, and both users can see each others model/video and hear each others audio
+  - [ ] State changes (name changes, A/V toggles) propagate between peers as expected
 
-### Testing Procedure
-Describe your testing procedure here. This can include unit tests, screen recordings of functionality, etc.
+### PR Specific Author Checklist
+What did you do to test your change in particular? For example, if your changes touches A/V functionality, how did you stress-test this part of the product?
+- [ ] xyz
+
+### Advanced Author Checklist
```
Comment (samjgorman) [inline .github/pull_request_template.md:20]: Perhaps within advanced author checklist we could add a 3 peers check?

File: .github/pull_request_template.md (hunk 2 above)
Comment (kanyesthaker) [reply .github/pull_request_template.md:12]: I think it's a lot of friction, i'm much less inclined to open 2 computers and have to sync my code between both. What's an example of a bug that shows up with 3+ peers that is not reproducible with 2? Might be forgetting something

File: .github/pull_request_template.md (hunk 3 above)
Comment (kanyesthaker) [reply .github/pull_request_template.md:20]: that could work!

**Review by samjgorman — APPROVED** [review]

File: .github/pull_request_template.md (hunk 2 above)
Comment (samjgorman) [reply .github/pull_request_template.md:12]: Sounds okay to start, note that you can test easier with 3 peers by using arc + chrome rather than 2 laptops

**Pull request merged**